docs/source/auth.rst

```diff
@@ -0,0 +1,566 @@
+
+Due to the nature of HTTP authentication, it is only supported on a remote Tracking Server, where users send
+requests to the server via REST APIs.
+
+How It Works
+============
+
+Built-in Admin User
+-------------------
+
+MLflow has a built-in admin user who has unrestricted access to all MLflow resources,
+including creating or deleting users, updating password and admin status of other users,
+and grating or revoking permissions from other users.
+The default admin user credential is as follow:
+
+.. list-table::
+   :widths: 10 10
+   :header-rows: 1
+
+   * - Username
+     - Password
+   * - ``admin``
+     - ``password``
+
+The admin will be created the first time MLflow authentication feature is enabled.
+It is recommended that you update the password as soon as possible after the admin is created.
+
+Multiple admin users can exist by promoting other users to admin.
+
+Permissions
+-----------
+
+The available permissions are:
+
+.. list-table::
+   :widths: 10 10 10 10 10
+   :header-rows: 1
+
+   * - Permission
+     - Can read
+     - Can update
+     - Can delete
+     - Can manage
+   * - ``READ``
+     - Yes
+     - No
+     - No
+     - No
+   * - ``EDIT``
+     - Yes
+     - Yes
+     - No
+     - No
+   * - ``MANAGE``
+     - Yes
+     - Yes
+     - Yes
+     - Yes
+   * - ``NO_PERMISSIONS``
+     - No
+     - No
+     - No
+     - No
+
+The default permission for all users is ``READ``. It can be changed in the :ref:`configuration <configuration>` file.
+
+Permissions can be set on two MLflow components: Experiments and Registered Models.
+Certain APIs can only be requested if the user has the required permission.
+Otherwise, a "Permission Denied" response will be returned.
+
+.. list-table::
+   :widths: 10 10 10 10
+   :header-rows: 1
+
+   * - API
+     - Endpoint
+     - Method
+     - Permission
+   * - :ref:`Create Experiment <mlflowMlflowServicecreateExperiment>`
+     - ``2.0/mlflow/experiments/create``
+     - ``POST``
+     - None
+   * - :ref:`Get Experiment <mlflowMlflowServicegetExperiment>`
+     - ``2.0/mlflow/experiments/get``
+     - ``GET``
+     - ExperimentPermission.can_read
+   * - :ref:`Get Experiment By Name <mlflowMlflowServicegetExperimentByName>`
+     - ``2.0/mlflow/experiments/get-by-name``
+     - ``GET``
+     - ExperimentPermission.can_read
+   * - :ref:`Delete Experiment <mlflowMlflowServicedeleteExperiment>`
+     - ``2.0/mlflow/experiments/delete``
+     - ``POST``
+     - ExperimentPermission.can_delete
+   * - :ref:`Restore Experiment <mlflowMlflowServicerestoreExperiment>`
+     - ``2.0/mlflow/experiments/restore``
+     - ``POST``
+     - ExperimentPermission.can_delete
+   * - :ref:`Update Experiment <mlflowMlflowServiceupdateExperiment>`
+     - ``2.0/mlflow/experiments/update``
+     - ``POST``
+     - ExperimentPermission.can_update
+   * - :ref:`Search Experiments <mlflowMlflowServicesearchExperiments>`
```

Comment:
a few endpoints including "search experiments" are not present in the REST API page, so the ref is not working

Reply:
We actually found `rest-api.rst` is outdated. Should be fixed soon.

Reply:
that's great! i'll keep an eye out

Reply:
Merged! Can you rebase?

Reply:
great! i added them back